src/main/scala/temple/generate/database/ast/ComparisonOperator.scala

```diff
@@ -0,0 +1,13 @@
+package temple.generate.database.ast
+
+/** AST representation for boolean comparisons */
+sealed trait ComparisonOperator
+
+object ComparisonOperator {
```

Comment:
Just a thought, is it worth implementing BETWEEN? I guess not, as it’s non-binary and would need a lot of rewriting for something that can be expressed in two inequalities.

Reply:
Yeah I think this would be nice for completeness but the required complexity probably isn't worth it at this point